cmd/openshift-install/main.go

```diff
@@ -102,4 +107,35 @@ func runRootCmd(cmd *cobra.Command, args []string) {
 	if err != nil {
 		logrus.Fatal(errors.Wrap(err, "invalid log-level"))
 	}
+
+	if version.CommitDate != "" {
```

Comment:
I do not think we should be putting this info on all commands but only on version command

Reply:
How often do folks run the `version` command?  I thought we'd want something that was gently in the face of anyone using an old installer, even if that's from a script or whatever.